machine/v1beta1/types_provider.go

```diff
@@ -224,6 +227,3 @@ type Condition struct {
 	// +optional
 	Message string `json:"message,omitempty"`
 }
-
-// Conditions provide observations of the operational state of a Machine API resource.
```

Comment:
This is probably on me, but did we intend to remove the Conditions type?? 

When im bringing this change into the MAO we're failing to build because this is missing.

Reply:
Yes we did, please substitute `Conditions` for `[]Condition` as you bring this into MAO